src/cascadia/PublicTerminalCore/Source.cpp

```diff
@@ -0,0 +1,291 @@
+#include "pch.h"
+#include "HwndTerminal.hpp"
+#include <DefaultSettings.h>
+#include "../../renderer/base/Renderer.hpp"
+#include "../../renderer/dx/DxRenderer.hpp"
+#include "../../cascadia/TerminalCore/Terminal.hpp"
+#include "../../cascadia/inc/cppwinrt_utils.h"
+#include "../../types/viewport.cpp"
+#include "..\..\types\inc\GlyphWidth.hpp"
+
+using namespace ::Microsoft::Terminal::Core;
+
+static WCHAR szTOOLBOXWNDCLASS[] = L"HwndTerminalClass";

```

Comment:
If it's going to be Hungarian, it needs to be correct. `wsz`. But really Hungarian is deprecated.

Also the `z` means we're enforcing null termination which only happens via the SAL macro in PCWSTR or PWSTR, not from a bare WCHAR.

Also we're generally doing `static constexpr std::wstring_view` for things like this these days.

Reply:
This variable is only used with functions that expect null-terminated c-style strings. Since `std::wstring_view` is not guaranteed to be null-terminated wouldn't it be better to leave this as a `WCHAR[]`, or perhaps a `std::wstring`?